installer-scripts/SwiftFormat.wxs

```diff
@@ -2,42 +2,41 @@
   <Package
       Language="1033"
       Manufacturer="nicklockwood"
-      Name="SwiftFormat for Windows x86_64"
+      Name="SwiftFormat for Windows $(var.ProductArchitecture)"
       UpgradeCode="98e01ac8-a17d-43fd-99ed-1cd8b58715bf"
       Version="$(var.ProductVersion)"
       Scope="perMachine">
-    <SummaryInformation Description="SwiftFormat for Windows x86_64" />
+    <SummaryInformation Description="SwiftFormat for Windows $(var.ProductArchitecture)" />
 
     <!-- NOTE(compnerd) use pre-3.0 schema for better compatibility. -->
     <Media Id="1" Cabinet="SwiftFormat.cab" EmbedCab="yes" />
 
     <!-- Directory Structure -->
+    <!-- Components -->
+
```

Comment:
I don't think that this is useful anymore as it is inlined.

```suggestion
```

Reply:
Done